### PR TITLE
Added jira_ignore_in_title

### PR DESCRIPTION
**Pull request by Qmando**
Adds a feature to ignore a field in a custom jira subject when searching for tickets.

### CODE REVIEW

File: elastalert/alerts.py [hunk 1]
```diff
@@ -361,9 +361,13 @@ def find_existing_ticket(self, matches):
         else:
             title = self.create_title(matches)
 
+        if 'jira_ignore_in_title' in self.rule:
+            title = title.replace(matches[0].get(self.rule['jira_ignore_in_title'], ''), '')
+
         # This is necessary for search for work. Other special characters and dashes
```
Comment (jjsendor) [inline elastalert/alerts.py]: `This is necessary for search *for* work` or `This is necessary for search *to* work`?

Comment (Qmando) [inline elastalert/alerts.py]: ++
catching typos that barely made it into the diff box :D

File: elastalert/alerts.py [hunk 2]
```diff
@@ -361,9 +361,13 @@ def find_existing_ticket(self, matches):
         else:
             title = self.create_title(matches)
 
-        # This is necessary for search for work. Other special characters and dashes
```
Comment (jjsendor) [inline elastalert/alerts.py]: Maybe _for search to work_?

Comment (Qmando) [inline elastalert/alerts.py]: This is just a preview for when elastalert works over the Tor network ;)

**Pull request merged**